docs/software/globus-mass-update-procedure.md

```diff
@@ -0,0 +1,185 @@
+    done
+```
+
+Tag `buildsys-macros` for the OSG release into the build tags:
+
+``` console
+[you@host]$ for el in el6 el7; do \
+       buildsys_macros_nvr=$(osg-koji -q list-tagged osg-3.4-$el-development \
+         buildsys-macros --latest | awk '{print $1}'); \
+       osg-koji tag-pkg $el-globus-build $buildsys_macros_nvr; \
+   done
+```
+
+Ensure Koji targets exist, one for each dver, e.g.:
+
+-   el6-globus (el6-globus-build &rarr; el6-globus)
+-   el7-globus (el7-globus-build &rarr; el7-globus)
+-   kojira-fake-el6-globus (el6-globus &rarr; kojira-fake)
+-   kojira-fake-el7-globus (el7-globus &rarr; kojira-fake)
+
+``` console
+[you@host]$ for el in el6 el7; do \
+       osg-koji add-target $el-globus $el-globus-build $el-globus; \
+       osg-koji add-target kojira-fake-$el-globus $el-globus kojira-fake; \
+   done
+```
+
+If basing the packages off of the Globus repos, add the Globus repos as external repos, and add them to the build tags (but not the dest tags).
+
+Edit `/etc/koji-hub/plugins/sign.conf` and set up the GPG signing for the RPMs. Run `/etc/koji-hub/plugins/fix-permissions` after editing the file.
+
+Per-package work
+----------------
+
+1.  cd into branches/globus
+2.  Download packages from <http://dl.fedoraproject.org/epel/6/SRPMS/>
+
+A useful alias:
+
+``` console
+[you@host]$ alias osg-build-globus="osg-build koji --ktt el6-globus --ktt el7-globus"
+```
+
+### Strict pass-through (no osg/ directory)
+
+1.  Run:
+
+        :::console
+        [you@uw]$ osg-import-srpm "%RED%<URL>%ENDCOLOR%"
+        [you@uw]$ osg-build-globus --scratch %RED%<PKG>%ENDCOLOR%
+
+
+2.  Commit - use a message like "Update to 3.12-1 from EPEL (SOFTWARE-2197)"
+
+3.  Do a non-scratch build.
+
+### Non-strict pass-through
+
+1.  Run:
+
+        :::console
+        [you@uw]$ osg-import-srpm --diff3 "%RED%<URL>%ENDCOLOR%"
+
+2.  Fix merge conflicts in the spec file. If not already there, put a .1 after the Release number to mark the changes as ours.
+3.  Run:
+
+        :::console
+        [you@uw]$ osg-build quilt %RED%<PKG>%ENDCOLOR%
+
+4.  Fix patches if necessary.
+5.  Run:
+
+        :::console
+        [you@uw>]$ osg-build-globus --scratch %RED%<PKG>%ENDCOLOR%
+
+6.  Commit - use a message like "Update to 8.29-1 from EPEL and merge OSG changes (SOFTWARE-2197)"
+
+7.  Do a non-scratch build.
+
+Testing
+-------
+
+Create a yum `.repo` file similar to `osg-minefield` that installs from the `el?-globus` repos. Enable this and `osg-minefield`.
+
+EL7 example:
+
+``` ini
+[globus]
+name=globus
+baseurl=http://koji.chtc.wisc.edu/mnt/koji/repos/el7-globus/latest/$basearch/
+failovermethod=priority
+priority=98
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-OSG
+consider_as_osg=yes
+```
+
+Merge
+-----
+
+### Koji (Mat/Carl)
+
+This requires a Koji administrator. Koji admins as of August 2017 are Mat Selmeci and Carl Edquist.
+
+1.  Untag broken versions that we don't want to ship.
+2.  Use `move-pkg`:
```

Comment:
This text can use some exposition: I can see that the command uses `move-pkgs` but it's not immediately clear why.

> Move packages to the development tags so we can continue with the normal development lifecycle

Or something